contrib/opencensus-ext-azure/opencensus/ext/azure/common/storage.py

```diff
@@ -170,3 +175,22 @@ def put(self, data, lease_period=0, silent=False):
             ),
         ))
         return blob.put(data, lease_period=lease_period, silent=silent)
+
+    def _check_storage_size(self):
+        size = 0
+        for dirpath, dirnames, filenames in os.walk(self.path):
+            for f in filenames:
+                fp = os.path.join(dirpath, f)
+                # skip if it is symbolic link
+                if not os.path.islink(fp):
+                    size += os.path.getsize(fp)
```

Comment:
It would be good to have try/except logic here in case something goes wrong reading the files